nginx.tmpl

```diff
@@ -72,6 +72,9 @@ upstream {{ $host }} {
 {{ end }}
 }
 
+{{ $default_host := or ($.Env.DEFAULT_HOST) "" }}
+{{ $default_server := index (dict $host "" $default_host "default_server") $host }}
```

Comment:
this is little bit hacky, but I didn't find another way to set a variable depending on another variable value.

Perhaps we could add a "when" function to docker-gen?
$default_server := when (eq $host $default_host) "default_server" ""